ext/nnstreamer/tensor_decoder/nnstreamer.fbs

```diff
@@ -0,0 +1,36 @@
+namespace NNStreamer;
+
+enum Tensor_type : int { 
+  NNS_INT32 = 0,
+  NNS_UINT32,
+  NNS_INT16,
+  NNS_UINT16,
+  NNS_INT8,
+  NNS_UINT8,
+  NNS_FLOAT64,
+  NNS_FLOAT32,
+  NNS_INT64,
+  NNS_UINT64 = 10,
+
+  NNS_END = 99
+  }
+
+struct frame_rate {
+  rate_n : int;
+  rate_d : int;
+}
+
+table Tensor {
+  name  : string;
+  type : Tensor_type = NNS_END;
+  dimension : string; // support up to 4th ranks.
+  data : [ubyte]; 
+}
+
+table Tensors {
+  num_tensor : int;
+  fr : frame_rate;
+  tensor : [Tensor]; // tensor size is limited to 16
+}
+
+root_type Tensors;
```

Comment:
Check your editor setting:) Do '''cat nnstreamer.fbs''' if you want to know what I am talking about. If fbs format does not allow a newline at the end of the file, just ignore this comment.

Reply:
New line has no problem with compilation. Thank you! Updated.